benchmark/buffers/buffer-copy.js

```diff
@@ -5,6 +5,12 @@ const bench = common.createBenchmark(main, {
   bytes: [0, 8, 128, 32 * 1024],
   partial: ['true', 'false'],
   n: [6e6]
+}, {
+  combinationFilter: (p) => {
+    return (p.partial === 'false' && p.bytes === 0) ||
+           (p.partial !== 'false' && p.bytes !== 0);
+  },
+  test: { partial: 'false', bytes: 0 },
```

Comment:
This doesn't seem intended.
```suggestion
```

Reply:
It is intended. It's either add this explicit configuration combination for the benchmark test or the order of the parameter values would need to be changed.

The reason for this is that benchmark tests pick the first value of each parameter for testing, which for this benchmark results in an empty benchmark queue (due to the combination filter). Without one of the two possible changes to fix this, the test/benchmark/test-benchmark-buffer.js test will fail.